src/styles/main.scss

```diff
@@ -1,3 +1,4 @@
-@import "blocks/page";
-@import "blocks/header";
-@import "blocks/catalog";
+@import "blocks/page.scss";
+@import "utils/_variables.scss";
```

Comment:
as i remember its better to put files from utils on the top

Reply:
Right. Fixed